cmd/kube-controller-manager/app/options/options.go

```diff
@@ -273,6 +275,7 @@ func (s *KubeControllerManagerOptions) Flags(allControllers []string, disabledBy
 	fs := fss.FlagSet("misc")
 	fs.StringVar(&s.Master, "master", s.Master, "The address of the Kubernetes API server (overrides any value in kubeconfig).")
 	fs.StringVar(&s.Generic.ClientConnection.Kubeconfig, "kubeconfig", s.Generic.ClientConnection.Kubeconfig, "Path to kubeconfig file with authorization and master location information (the master location can be overridden by the master flag).")
+	utilfeature.DefaultMutableFeatureGate.OverrideDefault(clientgofeaturegate.WatchList, true)
```

Comment:
here we actually don't know whether it should be enabled or disabled. 
it depends on the actual FG used by the binary.
in my case FG from `apiserver/pkg/features`

Reply:
I think we should actually call
```
// use our authoritative copy to replace client-go features
clientgofeaturegate.SetFeatureGates(utilfeature.DefaultMutableFeatureGate)
```